### PR TITLE
Add style checker

### PR DESCRIPTION
**Pull request by Hakuyume**
Related to #389.

### CODE REVIEW

**Comment by yuyu2172** [comment]
Thanks.
How about adding a doc string explaining the style check in the beginning of the script?
I am expecting the script to look something like below.

```python
"""
DOC
"""

import ...
```

**Comment by Hakuyume** [comment]
@yuyu2172 Nice suggestion. I added a docstring.

File: style_checker.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
+"""Coding style checker for ChainerCV
+
+This is a coding style checker used in ChainerCV.
+
+Usage:
+    $ python style_checker.py <directory path>
+
+This script checks following coding rules.
```
Comment (yuyu2172) [inline style_checker.py]: the following

**Comment by yuyu2172** [comment]
Great PR.
Looks good.

**Pull request merged**